.github/pull_request_template.md

```diff
@@ -18,21 +18,23 @@ Please explain in detail what the changes are in this PR and why they are needed
 Don't assume reviewers understand the original issue.
 
 ###  What is the type of the changes?
-
-Pick one of the following and delete the others:
-
-- New feature (a change which adds functionality)
-- Improvement (a change which is an improvement to an existing feature)
-- Bugfix (a change which fixes an issue)
-- Engineering (engineering change which doesn't change any feature or fix any issue)
-- Misc (other changes)
+<!--
+Please select the type of your changes:
+-->
+- [ ] New feature (a change which adds functionality)
```

Comment:
How about leave this unchanged, since it is a single selection and using checkboxes (which is usually a multi selection) doesn't fit well.

Reply:
I think this is a bit unfit.  But this will be more easier for contributors. 
So I think maybe we can keep it.

Reply:
Maybe we can include all in comments and let contributors to copy and paste one.

Reply:
fixed. PTAL~